test/journeys/server/data-api/space.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Cisco Spark Widget Test</title>
+  <meta charset="utf8">
+<link href="/dist/main.css" rel="stylesheet"></head>
+<body>
+<div style="width: 500px; height: 500px">
+  <main class="ciscospark-widget"
+    id="ciscospark-widget">
+
+</div>
+<script>
+  window.loadBundle = function (bundleFile) {
+    var script = document.createElement('script');
+    script.src = bundleFile;
+    document.head.appendChild(script);
+    script.onload = function () {
+      console.info(`bundle loaded`);
```

Comment:
Back ticks may be an issue in non chrome browsers. Should probably just keep everything in JS to single ticks.